### PR TITLE
feat(costmodel): add the cancel button to add rate form in wizard

### PR DESCRIPTION
**Pull request by boaz0**
related #1266 

- [x] Show the "Cancel" button when adding additional rate.
- [x] Change the "Save rate" button text to "Add rate"
- [x] Make "Add rate" and "Cancel" next to each other

### CODE REVIEW

**Comment by nlcwong** [comment]
Would it be possible to see a screenshot of the change?

**Comment by boaz0** [comment]
@nlcwong 
No rates:
image

Creating first rate:
image

Adding a second rate
image

image

**Comment by dlabrecq** [comment]
Sounds like @nlcwong requested changes?

**Comment by boaz0** [comment]
@dlabrecq - that's right.

**Comment by boaz0** [comment]
I will update the ticket but till a new change is pushed, DO NOT MERGE

File: package.json [hunk 1]
```diff
@@ -102,16 +102,20 @@
...
     "@babel/plugin-syntax-decorators": "^7.2.0",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
     "@babel/plugin-syntax-jsx": "^7.2.0",
     "@babel/plugin-syntax-typescript": "^7.2.0",
+    "@testing-library/jest-dom": "^5.0.2",
+    "@testing-library/react": "^9.4.0",
     "@types/enzyme": "^3.1.15",
     "@types/enzyme-adapter-react-16": "^1.0.3",
     "@types/jest": "^23.1.1",
+    "@xstate/test": "^0.3.0",
```
Comment (dlabrecq) [inline package.json]: We will want to start locking down the dependency versions. Instead of `"@xstate/test": "^0.3.0"`, simply use "0.3.0"

**Comment by nlcwong** [comment]
@boaz0 I'm a bit confused, I thought that when the user lands on the Price list step, the very first thing they would see is the rates table with an empty state in the middle:
EmptyTable

and then if they click on the "Add rate" button, they would get this screen:
AddRate

**Comment by boaz0** [comment]
image

image

image

**Comment by nlcwong** [comment]
@boaz0 I just noticed something in the screenshots, that I couldn't have noticed on my phone, the wizard buttons, they need to be disabled too while in "add rate" mode.  It is weird to have the back button and cancel button of the wizard to be enabled while having the cancel button enabled for the add rate mode.  Does that make sense?

**Comment by boaz0** [comment]
I can not disable the cancel/back buttons but I can hide them.

image

**Comment by dlabrecq** [comment]
@nlcwong I don't believe we should disable or hide the cancel and back buttons. It feels very odd that I cannot go back/cancel unless we finish adding a rate.

**Comment by nlcwong** [comment]
PF comment was to hide the Next button as well.  In a later issue, I will request we use a drawer component to fit all this into (will have to mock up a design for that).

**Comment by boaz0** [comment]
@nlcwong Done
image

**Pull request merged**